docs/tutorials/otio-filebundles.md

```diff
@@ -2,91 +2,122 @@
 
 ## Overview
 
-This document describes OpenTimelineIO's file bundle formats, otiod and otioz.  The intent is that they make it easier to package and send or archive OpenTimelineIO data and associated media.
+This document describes OpenTimelineIO's file bundle formats, `otiod` and `otioz`, as well as how to use the internal adapters to generate file bundles.  
 
-## Source Timeline
+The OTIOZ/D File Bundle format packages OpenTimelineIO data and associated media into a single file.  This can be useful for sending, archiving and interchange.
 
-For creating otio bundles, an OTIO file is used as input, whose media references are composed only of `ExternalReference` that have a target_url field pointing at a media file with a unique basename, because file bundles have a flat namespace for media.  For example, if there are media references that point at:
+## OTIOZ/D File Bundle Format Details
 
-`/project_a/academy_leader.mov`
+File bundles have two separate encodings, OTIOZ and OTIOD.  OTIOD is an encoding that uses a directory hierarchy of files, and OTIOZ is the identical structure packed into a single zipfile, currently using the python `zipfile` library.  In either case they will contain a content.otio which is the cut information.
 
-and:
+### Structure
 
-`/project_b/academy_leader.mov`
+File bundles have a consistent structure:
 
-Because the basename of both files is `academy_leader.mov`, this will be an error.  The adapters have different policies for how to handle media references.  See below for more information.
+OTIOD:
 
-### URL Format
+```
+something.otiod (directory)
+├── content.otio (file)
+└── media (directory)
+    ├── media1 (file)
+    ├── media2 (file)
+    └── media3 (file)
+```
 
-The file bundle adapters expect the `target_url` field of the `media_reference` to be in one of two forms (as produced by python's urlparse library):
+OTIOZ (adds the version.txt file and is encoded in a zipfile):
```

Comment:
Maybe add a note here that the `content.otio` and `version.txt` files are compressed and the media is uncompressed?

Reply:
good suggestion!